ui/Screen/Onboarding/EnterPassphrase.svelte

```diff
@@ -0,0 +1,171 @@
+<script>
+  import { createEventDispatcher } from "svelte";
+  import validatejs from "validate.js";
+  import {
+    ValidationStatus,
+    getValidationState,
+  } from "../../src/validation.ts";
+
+  import { Button, Input } from "../../DesignSystem/Primitive";
+
+  const dispatch = createEventDispatcher();
+
+  let repeatedPassphraseInput;
+  let passphrase;
+  let repeatedPassphrase;
+
+  let validations = false;
+  let beginValidation = false;
+
+  validatejs.options = {
+    fullMessages: false,
+  };
+
+  validatejs.validators.optional = (value, options) => {
```

Comment:
If we move validation logic out of the component, we can unit test it & also maintain consistency in how we validate

Reply:
Converted the `NameEntry` component.
Couldn't do it for `PasswordEntry` because the validation fields there depend on each other and our current `validations.ts` wrapper doesn't support that.